src/sentry/incidents/endpoints/organization_alert_rule_index.py

```diff
@@ -46,6 +55,41 @@ def get(self, request, organization):
             status__in=[RuleStatus.ACTIVE, RuleStatus.INACTIVE], project__in=project_ids
         )
 
+        teams = request.GET.getlist("team", [])
```

Comment:
I know this list will be small, but since we're doing `in` lets convert it to a `set`

Reply:
Added `teams = set(map(int, teams))` before the query, and after removing the strings from the list.

Reply:
I meant to do it at this step, so that we're not doing full list scans for each of your checks (like `"myteams" in teams`)